examples/rotating-cube/cube_mesh.zig

```diff
@@ -0,0 +1,49 @@
+pub const Vertex = extern struct {
```

Comment:
FYI I am against starting to duplicate all example code in this repository, that is not a good approach. We should start testing dusk against mach-core instead.